src/harness/testcases/WINNF_FT_S_IPR_testcase.py

```diff
@@ -251,8 +251,8 @@ def generate_IPR_2_default_config(self, filename):
     device_a['installationParam']['longitude'] = -88.09350
     device_b = json.load(
         open(os.path.join('testcases', 'testdata', 'device_b.json')))
-    device_b['installationParam']['latitude'] = 30.71570
-    device_b['installationParam']['longitude'] = -88.09350
+    device_b['installationParam']['latitude'] = 30.71571
+    device_b['installationParam']['longitude'] = -88.09351
 
```

Comment:
Just a question: why are we making such small changes on CBSD location ?

Reply:
There's no minimum distance between CBSDs.